sentinelhub/data_sources.py

```diff
@@ -0,0 +1,522 @@
+"""
+Module defining data sources
+"""
+import re
+import warnings
+from enum import Enum, EnumMeta
+from typing import Tuple
+from dataclasses import dataclass, asdict
+
+from aenum import extend_enum
+
+from .config import SHConfig
+from .constants import ServiceUrl
+from .exceptions import SHDeprecationWarning
+
+
+class _Source:
+    """ Types of satellite sources
```

Comment:
The docstring could be removed, or should just say "Sentinel Hub collections" (collections being the term SH team is using).

Reply:
When I change `DataSource` to `DataCollection` I renamed this to `_CollectionType`.